gradle/run.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+
+# Build a JAR of updated Android models for a schema.
+# Takes three command-line arguments:
+# 1 - UUID of schema
+# 2 - the schema, as a string
+
+# use temp file for output jar file and temp dir for input schemas
+MODEL_JAR=`mktemp data/models.XXXXX.jar`
+SCHEMA_SOURCE_DIR=`mktemp -d data/schemas.XXXXX`
+ANDROID_KEYSTORE="data/driver.keystore"
+KEYSTORE_ALIAS="driver"
+TSA_URL="http://timestamp.digicert.com"
+# name of json file gets used by jsonschema2pojo for the base class
+DRIVER_SCHEMA_FILE="DriverSchema.json"
+
+# clean up temporary files on exit
+trap "echo 'Cleaning up...'; rm $MODEL_JAR; rm -rf $SCHEMA_SOURCE_DIR; echo 'All done!'" EXIT
+
+set -e
+
+# delete redis key for schema, if it exists
+redis-cli -h $DRIVER_REDIS_HOST -p $DRIVER_REDIS_PORT -n $DRIVER_REDIS_JAR_DB DEL $1
```

Comment:
If there were an automated-retry process somewhere, then yes. I don't see this occurring in practice though.

The message queue is processed serially, so the subscriber task could re-check for an extant key before calling the bash script.  I'll do that then.